content/SCALE/SCALEUIReference/Storage/DevicesScreensSCALE.md

```diff
@@ -104,4 +104,8 @@ The system prevents losing existing data by stopping the add operation for the n
 
 **Force** overrides the safety check and adds the disk to the pool. Selecting this option erases any data stored on the disk!
 
+**Preserve Power Management and S.M.A.R.T. settings** transfers all power management and S.M.A.R.T. test configurations from the original disk to your replacement disk. The dialog window automatically selects this option, but you can uncheck it if you wish to reset your configurations.
+
+**Preserve disk description** maintains any descriptions associated with the original disk, which prevents you from needing to copy descriptors to the new disk manually. The dialog window automatically selects this option, but you can uncheck it if you want the replacement disk to use descriptors which differ from those attached to the original disk.
+
```

Comment:
This is a duplication of the text above. When written for UI ref, we want to just say what the setting is about or does not how to use it. I know our UI ref content need cleaning up but here you can change this content to be consistent with documenting UI ref content. How about this instead?
```suggestion
**Preserve Power Management and S.M.A.R.T. settings** transfers all power management and S.M.A.R.T. test configurations from the original disk to your replacement disk. Selected by default. Disable to reset your configuration. 

**Preserve disk description** maintains any descriptions associated with the original disk. Selected by default. Disable to change the replacement disk descriptors attached to the original disk.
```